### ISSUE TITLE
Feature Request: Toggle Joystic Gestures Only

### ISSUE THREAD

**Issue by minusreality** [alvr-org/ALVR#2196]
When using a controller with joysticks, the specific gestures related to joysticks will conflict with the physical controller joysticks causing erratic behavior. I'd like to be able to disable only the joystick gestures so that I can use my controller joysticks for locomotion and my hands for everything else. Would this be possible?

**Comment by chrisssj2** [comment]
Yes I experience this too, needed to game with a blanket over my hands. till i realized i need to turn off gestures lmao...
would be great if it all just worked together...
